donation/forms.py

```diff
@@ -10,7 +10,14 @@ class FilterDonationForm(forms.Form):
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self.method = "get"
+
+        # Asignamos los valores de los filtros como valores iniciales
```

Comment:
Try to put the comments in English, I say it more than anything because of the defined methodology.